src/main/java/cam72cam/immersiverailroading/thirdparty/event/GetTagEvent.java

```diff
@@ -0,0 +1,10 @@
+package cam72cam.immersiverailroading.thirdparty.event;
+
+import java.util.UUID;
+
+public class GetTagEvent extends TagEvent {
```

Comment:
These classes can be moved to static classes inside of TagEvent, that's how forge does it a lot of the time.  Other than that, this PR looks good!

Reply:
Done!